docs/CS/CLAIMSET-MGMT.md

```diff
@@ -0,0 +1,191 @@
+# Claimset Management
+
+Ed-Fi API security relies on the concept of a claimset, which groups together
+the operations allowed by an API client in the following dimensions:
+
+* Authorization Strategy (namespace, ownership, various relationships, or no
+  restriction).
+* Resource - all of the entities in both the Resource API and the Descriptors
+  API
+  * There is also a hierarchy: for example, grouping some Descriptors together
+    as "system descriptors" or "managed descriptors", or grouping together all
+    education organizations.
+* Actions (create, read, update, delete, and read changes (change queries)).
+
+## Legacy Approach
+
+### Database Model
+
+In the ODS/API and the Admin API applications, these data are stored in the
+`EdFi_Security` database with the following model:
+
+```mermaid
+erDiagram
+  Action ||--o{ ClaimSetResourceClaimActions : has
+  AuthorizationStrategies ||--o{ ResourceClaimActionAuthorizationStrategies : has
+  AuthorizationStrategies ||--o{ClaimSetResourceClaimActionAuthorizationStrategyOverrides : has
+  ClaimSetResourceClaimActions ||--o{ ClaimSetResourceClaimActionAuthorizationStrategyOverrides : has
+  ClaimSets ||--o{ ClaimSetResourceClaimActions : has
+  ResourceClaimActions ||--o{ ResourceClaimActionAuthorizationStrategies : has
+  ResourceClaims ||--o{ ClaimSetResourceClaimActions : has
+  ResourceClaims ||--o{ ResourceClaims : has
+  ResourceClaims ||--o{ ResourceClaimActions : has
+```
+
+### Admin API REST Interface
+
+The Admin API interface supports the following operations related to these tables:
+
+* /action
+  * GET all
+* /authorizationStrategies
+  * GET all
+* /claimsets
+  * Regular CRUD operations:
+    * GET all
+    * GET by id
+    * POST
+    * PUT
+    * DELETE
+  * CLOB ("Character Large OBject") operations, which unifies all of the dimensions of a Claimset into one payload:
+    * GET /export
+    * POST /copy
+    * POST /import
+  * /claimsets/{id}/resourceClaimActions
+    * POST
+    * PUT
+    * DELETE
+    * /claimsets/{id}/resourceClaimActions/{id}
+      * POST overrideAuthorizationStrategy
+      * POST resetAuthorizationStrategies
+* /resourceClaims
+  * GET all
+  * GET by id
```

Comment:
Are we really sure that the CLOB payload includes all of the child resource claims? For example, in the "managed descriptors" payload, does it really include a list of all of the descriptors?